.github/workflows/labeler.yml

```diff
@@ -13,10 +13,19 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 5
 
+    # Required by gh
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      PR_NUMBER: ${{ github.event.number }}
+
     steps:
     - uses: actions/labeler@v4
       with:
         # Auto-include paths starting with dot (e.g. .github)
         dot: true
         # Remove labels when matching files are reverted or no longer changed by the PR
         sync-labels: true
+
+    # Apply OS-windows label if PR title contains 'Windows'
+    - run: gh pr edit $PR_NUMBER --add-label OS-windows
+      if: contains(github.event.pull_request.title, 'Windows')
```

Comment:
note this is case sensitive

Reply:
It should not according to https://docs.github.com/en/actions/learn-github-actions/expressions#contains

<img width="749" alt="image" src="https://github.com/containerd/rust-extensions/assets/865334/48233292-8d95-4726-aadf-9246b9f1769f">